### ISSUE TITLE
What is best way to make all tables sortable by columns in docsy?

### ISSUE THREAD

**Issue by laszewsk** [google/docsy#604]
What is best way to make all tables sortable by columns in docsy while using the original markdown tables without any special  tags around the table?

**Comment by LisaFC** [comment]
We don't have anything that would do that in the theme right now - you'd need to add your own Javascript/CSS. Something like this might work? http://tristen.ca/tablesort/demo/

**Comment by mahi072** [comment]
Hi I would like to work on this issue.
Please assign me this.

**Comment by emckean** [comment]
Hi @mahi072 we don't assign issues in this project. If you would like to contribute please check out our [Contributing Guidelines](https://www.docsy.dev/docs/contribution-guidelines/).

**Comment by mahi072** [comment]
Hi @laszewsk  and @emckean , Can you please elaborate the issue.
Is there any documentation requirement in this?

**Comment by laszewsk** [comment]
when doing a table in markdown it typically does not have the ability to sort by column.
we need a way to add this feature. This could be done through some n=enhancement or some documentation changing the CSS.  The documentation should be simple and should include a working example. The goal is that someone without knowledge in CSS can set this up in 5 minutes or less and it is idealy done automatically. There should be a feature to switch this on and off

**Comment by mahi072** [comment]
Thanks @laszewsk for explaining this.
Also Can you please tell me the location in directory where this script/feature to be added ?

**Comment by mahi072** [comment]
Hi @laszewsk, @LisaFC 
if I'm clear then you want that all the tables present in docsy which is in this form(markdown) should be able to sort by column.(please see below image)
![image](https://user-images.githubusercontent.com/89216134/220374270-94e5ba22-9419-4ded-8875-7bff4a1ae94a.png)
 
And for this you want a documentation which contains a CSS script that can be used further to add and implement the sorting. 

please correct me if m wrong anywhere, and let me know where exactly in the directory I can make the changes if possible ?

**Comment by laszewsk** [comment]
yes. something similar to this: https://datatables.net/examples/basic_init/table_sorting.html 
I do not need paging, just sorting would be nice on individual columns.

**Comment by tiffany76** [comment]
[OpenTelemetry](https://github.com/open-telemetry) contributor here. Just up-voting this issue. Our project could benefit from a column-sort feature.
